docs/troubleshooting.md

```diff
@@ -23,7 +33,7 @@ Output should show all checkboxes:
 -> type(cosmos) key(✔) bal(✔) path(✔)
 ```
 
-## **Verify valid `chain`, `client`, and `connection`**
+### **Verify valid `chain`, `client`, and `connection`**
```

Comment:
This line seems like it should read something along the lines of, "Verify valid `Clients`, `Connections`, and `Channels`"

Reply:
I had to double check this too, but this should indeed be `chain`.
It queries the latest height. If it returns an error, then chns(✘).

Added an extra note about this.